<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edicao</class>
 <widget class="QMainWindow" name="Edicao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lblURL">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtURL">
       <property name="toolTip">
        <string>URL do curso</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	color: blue;
	text-decoration: underline;
	background-color: white;
}</string>
       </property>
       <property name="placeholderText">
        <string>https://youtube.com</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="lblPreco">
       <property name="text">
        <string>Preço</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="spinPreco">
       <property name="toolTip">
        <string>Preço do curso</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QDoubleSpinBox {
	background-color: white;
}</string>
       </property>
       <property name="prefix">
        <string>R$ </string>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::StepType::DefaultStepType</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblNome">
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNome">
       <property name="toolTip">
        <string>Nome do curso</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: white;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_3">
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateInicio">
       <property name="toolTip">
        <string>Data de inicio</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QDateEdit {
	background-color: white;
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblInicio">
       <property name="text">
        <string>Inicio</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>489</y>
      <width>731</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Editar curso</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #0000FF;
	color: white;
	font-weight: bold;
	padding-left: 40px;
	padding-right: 40px;
}

QPushButton:hover {
	border: 1.2px solid black;
}</string>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Resetar valores</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #f3ff05;
	color: white;
	font-weight: bold;
	padding-left: 40px;
	padding-right: 40px;
}

QPushButton:hover {
	border: 1.2px solid black;
}</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>731</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descrição</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="txtDescricao">
       <property name="toolTip">
        <string>Descrição do curso</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
	background-color: white;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkCompleto">
       <property name="toolTip">
        <string>Conclusão do curso</string>
       </property>
       <property name="text">
        <string>Está completo?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-weight: bold;
	font-size: 36px;
}</string>
    </property>
    <property name="text">
     <string>Edição</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
